api/app/serializers/v1/feature_serializer.rb

```diff
@@ -5,32 +5,31 @@ class FeatureSerializer < ManifoldSerializer
 
     abilities
 
-    typed_attribute :id, NilClass
-    typed_attribute :header, NilClass
-    typed_attribute :header_formatted, NilClass
-    typed_attribute :subheader, NilClass
-    typed_attribute :subheader_formatted, NilClass
-    typed_attribute :body, NilClass
-    typed_attribute :body_formatted, NilClass
-    typed_attribute :link_text, NilClass
-    typed_attribute :link_url, NilClass
-    typed_attribute :link_target, NilClass
-    typed_attribute :style, NilClass
-    typed_attribute :hidden, NilClass
-    typed_attribute :background_color, NilClass
-    typed_attribute :foreground_color, NilClass
-    typed_attribute :header_color, NilClass
-    typed_attribute :layout, NilClass
-    typed_attribute :created_at, NilClass
-    typed_attribute :updated_at, NilClass
-    typed_attribute :foreground_top, NilClass
-    typed_attribute :foreground_left, NilClass
-    typed_attribute :foreground_position, NilClass
-    typed_attribute :position, NilClass
-    typed_attribute :live, NilClass
-    typed_attribute :include_sign_up, NilClass
-    typed_attribute :foreground_styles, Types::Hash
-    typed_attribute :background_styles, Types::Hash
+    typed_attribute :header, Types::String
+    typed_attribute :header_formatted, Types::String.meta(read_only: true)
+    typed_attribute :subheader, Types::String.optional
+    typed_attribute :subheader_formatted, Types::String.optional.meta(read_only: true)
+    typed_attribute :body, Types::String
+    typed_attribute :body_formatted, Types::String.meta(read_only: true)
+    typed_attribute :link_text, Types::String
+    typed_attribute :link_url, Types::Serializer::URL
+    typed_attribute :link_target, Types::String.optional # TODO: Add description for what this does
```

Comment:
I do not think I understand how a link target differs from a link url.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target

Reply:
Thank you. I added that to the description of the attribute.